src/app/wallets/send-on-chain.ts

```diff
@@ -241,7 +241,7 @@ const executePaymentViaIntraledger = async ({
             recipientWalletId: recipientWallet.id,
             recipientWalletCurrency: recipientWallet.currency,
             recipientUsername: recipientAccount.username,
-            memoPayer: memo ?? null,
+            memoPayer: memo ?? undefined,
```

Comment:
are you sure we want to exclude `""`? else please use `||`

Reply:
I think it makes sense to exclude ```""```. Can you think of any scenario where we would want it?